corehq/apps/app_manager/templates/app_manager/v2/managed_app.html

```diff
@@ -89,12 +89,19 @@ <h2 class="appnav-name">
           <button type="button" class="popover-additem-option new-module" data-type="survey">
             <i class="fa fa-file-o"></i>
             {% trans "Surveys" %}
+            <p>{% blocktrans %}
+              <i class="fa fa-info-circle"></i> Collect data<br/>once.
```

Comment:
Minor: the `<br />` tag might be weird for translations (this might not be two lines in another language).